lesson-6/twenty-one.js

```diff
@@ -79,29 +93,58 @@ function addTotal(hand) {
     }
   });
 
-  hand.filter(value => value === 'A').forEach(_ => {
-    if (total > POINT_LIMIT) total -= 10;
+  hand.forEach(card => {
+    if (card[1] === 'A' && total > POINT_LIMIT) total -= 10;
   });
 
   return total;
 }
 
-
 function busted(handTotal) {
   return handTotal > POINT_LIMIT;
 }
 
-function displayRoundScore(playerHand, dealerHand) {
-  prompt(`Player's final hand: ${displayHand(playerHand)}
+function playerTurn(deck, playerHand, dealerHand, playerPoints) {
```

Comment:
code extracted from innermost while loop of original file (player hits or stays)